include/deal.II/lac/trilinos_solver.h

```diff
@@ -559,6 +574,23 @@ namespace TrilinosWrappers
     solve(dealii::LinearAlgebra::distributed::Vector<double>       &x,
           const dealii::LinearAlgebra::distributed::Vector<double> &b);
 
+    /**
+     * Solve the linear system <tt>Ax=b</tt> based on the
+     * package set in the constructor or initialize(). Note the matrix is not
+     * refactorized during this call.
```

Comment:
```suggestion
     * refactored during this call.
```

Reply:
I applied this change to the old functions as well :+1: